.github/workflows/deploy-react-storybook.yml

```diff
@@ -0,0 +1,54 @@
+name: Deploy React storybook to IBM Cloud
+
+on:
+  push:
+    tags:
+      # Matches tags that have the shape `vX.Y.Z`. This will also match tags
+      # with a preid, for example `vX.Y.Z-rc.0`. Reference:
```

Comment:
```suggestion
      # with a period, for example `vX.Y.Z-rc.0`. Reference:
```

typo 😄

Reply:
What I was trying to show was that it could support the `preid` after `vX.Y.Z`, specifically the `-rc.0` bit 👀

Reply:
haha oh ignore me